TACC/Core-CMS-Custom
# task/WP-212 clear notes excep exten edit modal

## Overview

To allow the notes field in the edit modal of both the admin exception and extension page to be cleared.

## Related

- [WP-212](https://jira.tacc.utexas.edu/browse/WP-212)


## Changes

Updated queries that update exception and extension records so that notes can be cleared from the front end.

## Testing

1. Go to [http://localhost:8000/administration/list-exceptions/](http://localhost:8000/administration/list-exceptions/ )and select Edit Record from Action drop down
2. Find a record with notes already in the edit modal. Try deleting all of the text in the notes field and submitting. 
3. Make sure there's no errors. Open the modal for the same record and check that the the notes section is now empty.
4. Go to [http://localhost:8000/administration/list-extensions/](http://localhost:8000/administration/list-extensions/and) select Edit Record from Action drop down
5. Find a record with notes already in the edit modal. Try deleting all of the text in the notes field and submitting. 
6. Make sure there's no errors. Open the modal for the same record and check that the the notes section is now empty.
